releasenotes/README.md

```diff
@@ -5,6 +5,22 @@ Notes should be created as part of the pull request for any user facing changes.
 a release, the release notes utility will be run in order to generate a release notes file
 which will be reviewed by the release managers and documentation team.
 
+## When to Add Release Notes
+
+Release notes should be added for any user facing changes. These include:
+* CLI changes
+* API changes
+* Configuration schema change
+* Behavioral change
+* Change in non-functional attributes such as efficiency or availability, availability of a new platform
+* A warning about a deprecation
+* Fix of a previous Known Issue
```

Comment:
What about feature promotion?  would be good to have it in release note.
Ennablement of previously dark launched feature, could be already covered under "behavioral change" but may worth mentioning it.

Reply:
What about security fixes?

Reply:
I think adding feature promotion here works. I was considering calling them out in a separate section highlighting promoted features, but adding them as an item in the section they belong to might work better. 

As far as security fixes, they go in the securityNotes section.